app/code/Magento/Catalog/Block/Product/View/Attributes.php

```diff
@@ -85,8 +86,8 @@ public function getAdditionalData(array $excludeAttr = [])
                 } elseif ($attribute->getFrontendInput() == 'price' && is_string($value)) {
                     $value = $this->priceCurrency->convertAndFormat($value);
                 }
-
-                if (is_string($value) && strlen($value)) {
+    
+                if ($value instanceof Phrase || (is_string($value) && strlen($value))) {
```

Comment:
@maghamed, right part of `(is_string($value) && strlen($value))` is not checked in case of `Phrase` currently.

I didn't think deeper if this is a problem or not (like, maybe there is no need to check length at all).